components/api/Cargo.toml

```diff
@@ -23,11 +23,10 @@ chrono = { version = "0.4.6", features = ["serde"] }
 failure = "0.1.5"
 futures = "0.1.25"
 log = "0.4.6"
-serde = "1.0.101"
+serde = { version = "1.0.101", features = ["derive"] }
```

Comment:
Is it necessary in our code base or just more convenient than before?

Reply:
I had errors without it before, but now it works. I'll remove it.